src/components/FullPlayground.tsx

```diff
@@ -756,59 +727,59 @@ export function ThemedAppView(props: { datastore: DataStore }) {
               )}
             </div>
             {AppConfig().discord.inviteUrl ? (
-              <Button
-                className={classes.hideTextOnMed}
-                size="small"
-                href={AppConfig().discord.inviteUrl}
-                startIcon={
+              <Button asChild variant="link" size="sm">
```

Comment:
`asChild` retains the styling of a button but renders the child as the primary element, which makes links behave as expected.

the `link` variant gives some nice styling. I don't think we should use this for all links, but it looks nice and works well for the buttonlinks we've got.